assets/src/scripts/utils.js

```diff
@@ -45,23 +45,6 @@ export const replaceHtmlEntities = function (someString) {
     return someString;
 };
 
-/**
- * Calculate the difference in days between two Date objects
- *
- * @param date1
- * @param date2
- * @returns {number}
- */
-export const deltaDays = function (date1, date2) {
```

Comment:
Removed because we no longe use this utility method and the luxon library has equivalent capability.